temporalio/workflow.py

```diff
@@ -192,11 +206,33 @@ def signal(
     ...
 
 
+@overload
+def signal(
+    *, unfinished_handlers_policy: UnfinishedHandlersPolicy
```

Comment:
```suggestion
    *, unfinished_policy: UnfinishedHandlersPolicy
```

Would be ok stripping `handlers` from the option name (option name still mostly matches enum name). Also, I wonder if `HandlerUnfinishedPolicy` therefore is a better enum name